src/components/Carousel/Carousel.scss

```diff
@@ -58,14 +65,18 @@
   border-radius: 24px;
   background-color: white;
   border: 1px solid #ddd;
+  cursor: pointer;
+  &:focus {
+    outline: none;
+  }
 }
 
 .left-arrow {
-  left: 24px;
+  left: 5px;
 }
 
 .right-arrow {
-  right: 24px;
+  right: 5px;
 }
```

Comment:
As per your suggestion not use px, please convert it rem

Reply:
Done